src/app/activityindicator/activityindicator.component.scss

```diff
@@ -0,0 +1,18 @@
+.indicator{
```

Comment:
this isn't the [scss style](http://thesassway.com/beginner/how-to-structure-a-sass-project) we agreed upon.